test/runtests.jl

```diff
@@ -63,11 +63,6 @@ facts("Tests: dot{x}=x^2, x(0) = 3; nsteps <= maxsteps") do
     @fact length(tv) --> 14
     @fact length(xv[:,1]) --> 14
     @fact length(xv[:,2]) --> 14
-    if VERSION < v"0.5-"
-        @fact xv[1,1:end] --> q0'
-    else
-        @fact xv[1,1:end] --> q0
-    end
```

Comment:
Moreover: in `TaylorIntegration.jl` there is the line `import Compat.view` which should also be removed...